rust/src/main.rs

```diff
@@ -133,9 +135,11 @@ fn main() -> anyhow::Result<()> {
 }
 
 /// Run provided test cases and filter according to features and flags availability.
+//TODO: Refactor this function
 fn run_test_cases(
     test_cases: &Vec<&TestCase>,
     verbose: bool,
+    is_root: bool,
```

Comment:
Why take this as an argument, rather than just looking up the UID from within this function?

Reply:
This was just a mistake.